src/lib/comparator.mli

```diff
@@ -4,9 +4,14 @@ val make : ('a -> 'a -> int) -> (Format.formatter -> 'a -> unit) -> 'a t
 val compare : 'a t -> 'a -> 'a -> int
 val pp : 'a t -> Format.formatter -> 'a -> unit
 val int : int t
+val int32 : int32 t
+val int64 : int64 t
 val float : float t
 val bool : bool t
+val char : char t
 val string : string t
+val bytes : bytes t
+val unit : unit t
```

Comment:
`unit` is perhaps not terrible useful but I'm fine to keep for completeness :)

Reply:
Yep, `unit` by itself is probably not that useful. However, when using in combination with `result` and `option` type it is quite useful, for example,
```
type unit_result = (unit, string) result [@@deriving show, ord, popper]
type unit_opt = unit option [@@deriving show, ord, popper]
```